jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -38,7 +38,8 @@ Author: Daniel Kroening, kroening@kroening.com
   "(java-max-input-tree-depth):" /* will go away */ \
   "(max-nondet-tree-depth):" \
   "(java-max-vla-length):" \
-  "(java-cp-include-files):" \
+  "(load-containing-class-only)" \
+  "(java-cp-include-files)" \
```

Comment:
Missing `:` (unless you really wanted to change the syntax here, which isn't reflected in the code.

Reply:
Typo. Fixed.